### ISSUE TITLE
does not work on vue2 with syntax in Readme

### ISSUE THREAD

**Issue by vesper8** [permafrost-dev/vue-ray#10]
This is an odd one because when I first approached you about this vue2 problem you told me the solution was to do 

```
import { RayPlugin } from 'vue-ray/vue2';

Vue.use(RayPlugin.default, {
```

instead of

```
import { RayPlugin } from 'vue-ray/vue2';

Vue.use(RayPlugin, {
```

Which worked at the time, and then you solved it and it was no longer needed.. and for a while I was able to test `vue-ray` without adding the `.default`

But today (using latest versions `vue-ray 1.16` and `node-ray 1.13`) the problem started happening again.

It would correctly intercept console.log events (which I opened another issue about because I don't want that), but if I tried to use `this.$ray` it would throw an error and say that `this.$ray` is not defined

When I re-added the `.default` then it started working correctly again

**Comment by patinthehat** [comment]
@vesper8 Thank you, I'm looking into this now.

**Comment by patinthehat** [comment]
@vesper8 This should be resolved as of version `1.17.1`.  The following should work for you with vue 2:

```js
// main.js

Vue.config.productionTip = false

import Vue from 'vue';
import { RayPlugin } from 'vue-ray/vue2';
import App from './App.vue'

Vue.use(RayPlugin, {
  interceptErrors: true,
  host: 'localhost',
  port: 23517,
  showComponentEvents: [
    'created',
    'mounted',
  ],
  nodeRaySettings: {
    interceptConsoleLog: true,
  }
});

new Vue({
  render: h => h(App),
}).$mount('#app');

console.log('loaded');
```
